### PR TITLE
Use general rotation function to rotate pixels

### LINKED ISSUE #438: check if model/camera.py::_rotate_pixels() can use the util.general.rotate function
As discussed before with @Orel, we want to check if `model/camera.py::_rotate_pixels()` can be adjusted to use `util.general.rotate` function. One only rotation function should be enough for all the necessary rotations in the code.

       The function seems to be generic (it is in `general.py`), but here it is about telescope positions? Why e.g., can't we replace  `model/camera.py::_rotate_pixels()` and have only one rotation function?

_Originally posted by @GernotMaier in https://github.com/gammasim/gammasim-tools/pull/429#discussion_r1048465789_

### PR DESCRIPTION
**Pull request by orelgueta**
Fixes #438.

### CODE REVIEW

**Review by VictorBarbosaMartins — APPROVED** [review]
Hi Orel. Thanks for implementing this. now we have a generic function for rotation that can be used elsewhere. I am approving assuming that you had a look also in the images produced and confirmed, e.g. that the direction of rotation is what we expect it to be. I am also pushing a few changes to the docstrings.

File: simtools/model/camera.py
Comment (orelgueta) [reply simtools/model/camera.py:169]: Yes, it works because "rad" and u.rad are equivalent. However, since I have to make a change to the log, I will change this here as well in order to save the translation.

File: simtools/model/camera.py
Comment (orelgueta) [reply simtools/model/camera.py:171]: Right, I will remove the `.value`. I will also mention the direction of rotation is clockwise.

**Comment by orelgueta** [comment]
Thanks for the review. I looked at all images of the cameras of course and confirmed they are still correct. I will wait for the tests to pass after the change and if so I will merge.

**Pull request merged**
